src/Controls/src/Core/Handlers/Items/iOS/ItemsViewController.cs

```diff
@@ -124,7 +124,7 @@ void CheckForEmptySource()
 				// If we're going from empty to having stuff, it's possible that we've never actually measured
 				// a prototype cell and our itemSize or estimatedItemSize are wrong/unset
 				// So trigger a constraint update; if we need a measurement, that will make it happen
-				ItemsViewLayout.ConstrainTo(CollectionView.Bounds.Size);
+				ConstrainToItemsView();
```

Comment:
https://github.com/xamarin/Xamarin.Forms/pull/15289/files

Reply:
This one also reverted for now in Forms: https://github.com/xamarin/Xamarin.Forms/issues/15470

Reply:
Reverted